xgi/classes/hypergraph.py

```diff
@@ -935,6 +936,77 @@ def clear_edges(self):
         self._edge.clear()
         self._edge_attr.clear()
 
+    def merge_duplicate_edges(
+        self, rename="first", merge_rule="first", multiplicity=True
+    ):
+        """Merges edges which have the same members.
+
+        Parameters
+        ----------
+        rename : str, optional
+            Either "first", "tuple", or "new", by default "first"
+            If "first", the new edge ID is the first of the sorted
+            duplicate edge IDs. If "tuple", the new edge ID is a 
+            tuple of the sorted duplicate edge IDs. If "new", the
+            _edge_uid counter is selected and incremented.
```

Comment:
We should assume the user has no knowledge of `_edge_uid` so we shouldn't mention it.
```suggestion
            tuple of the sorted duplicate edge IDs. If "new", a new ID will be
            selected automatically.
```

Reply:
Fixed.